api4/integration_action_test.go

```diff
@@ -84,7 +83,7 @@ func TestOpenDialog(t *testing.T) {
 	pass, resp = Client.OpenInteractiveDialog(request)
 	CheckBadRequestStatus(t, resp)
 	assert.False(t, pass)
-}
```

Comment:
This test still causing issues?

Reply:
If you pushed a patch with fixes for this I might not have tested it. Will check.

Reply:
I didn't push any changes, so it's probably still broken then